content/hi/partners/_index.html

```diff
@@ -0,0 +1,91 @@
+---
+title: सहयोगी
+bigheader: कूबरनेट्स सहयोगी
+abstract: कूबरनेट्स इकोसिस्टम का विस्तार.
+class: gridPage
+cid: सहयोगी
+---
+
+<section id="users">
+    <main>
+        <h5>कुबरनेट्स सहयोगियों संग एक मज़बूत, जीवंत कोड बेस का निर्माण करता है जो संपूरक मंचों के स्पेक्ट्रम को सहयोग करता है।</h5>
```

Comment:
There are few words which are not making up the sentence.

Like `जीवंत` and  `संपूरक`

Reply:
I think both these words perfectly satisfy the words for this particular context.
Also, we can remove जीवंत from the sentence as the meaning will be the same either way.

Reply:
Yeah, you are right we can remote `जीवंत` doesn't change the meaning of the sentence.

The correct spellings of `संपूरक` are `संपूर्ण`.

Reply:
Okay I will remove जीवंत from the translation, also संपूरक means Complementary while संपूर्ण means Complete, these are both totally different words, so I will keep this as it is.

Reply:
It's just to make the sentence easier, it better to use `संपूर्ण`, otherwise using `संपूरक` is also not a bad choice

Reply:
Thanks.